src/frontend/src/pages/index.js

```diff
@@ -1,23 +1,48 @@
-import React from 'react';
-import { StaticQuery, graphql } from 'gatsby';
-
-import Layout from '../components/Layout/Layout';
+import React, { useEffect, useState } from 'react';
+import Header from '../components/Header';
+import Banner from '../components/Banner';
+import Posts from '../components/Posts';
+import ScrollToTop from '../components/ScrollToTop';
+import SEO from '../components/SEO';
```

Comment:
Should we move this to the `PageBase`?

Reply:
We should, don't know if we should do that here or in a follow up PR.

Reply:
Follow-up is fine.